src/OrderInventory.ts

```diff
@@ -0,0 +1,103 @@
+import { genericCleanup } from "./util";
+
+/**
+ * Sort Inventories by Code
+ * TODO: only grab the inventory where the button was pressed
+ */
+export class OrderInventory {
+  private sortByCode: boolean = false;
+  private tag = "sort-by-code";
+  private cleanups: Array<() => void> = [];
+
+  cleanup() {
+    this.cleanups.forEach((f, i) => {
+      f();
+      delete this.cleanups[i];
+    });
+    genericCleanup(this.tag);
+  }
+
+  run() {
+    this.sortInventoriesByCode();
+    this.addSortByCodeButton();
+  }
+
+
+
+  private sortInventoriesByCode() {
+    if (this.sortByCode) {
```

Comment:
if you invert this if with a return, you reduce the nesting in the following code and prevent potential errors with braces placed in the wrong line or similar.